scripts/main.py

```
@@ -6,17 +6,17 @@
 import smtplib
 
 # TODO figure out how to import these nicely and still have mypy work.
```

Pylint (3.3.7)

[warning] TODO figure out how to import these nicely and still have mypy work.

(W0511)